condition/condition_plugins/include/condition_plugins/reach_position_condition.h

```diff
@@ -1,6 +1,7 @@
 #ifndef CONDITION_PLUGINS_REACH_POSITION_CONDITION_H_INCLUDED
 #define CONDITION_PLUGINS_REACH_POSITION_CONDITION_H_INCLUDED
 
+#include <geometry_msgs/msg/pose_stamped.hpp>
```

Comment:
do you need `pose_stamped.hpp` or just `pose.hpp`? Below you just define an unstamped pose

Reply:
Fixed.